react/Button/Button.js

```diff
@@ -4,6 +4,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 import classnames from 'classnames';
+import { capitalize } from 'lodash';
```

Comment:
Heads up, this line is pulling in the entirety of lodash. Should be importing from `lodash/capitalize`.